languages/elixir/exercises/concept/basics/.docs/after.md

```diff
@@ -1,51 +1,75 @@
 
-Elixir is an [functional-programming language][functional-programming] and requires all named [functions][functions] to be defined in a [module][modules]. The `defmodule` keyword is used to define a module. All modules are available to all other modules at runtime and do not require an _access modifier_ to make them visible to other parts of the program. A [module][modules] is analogous to a _class_ in other programming languages.
+  - Named functions are defined with `def`.
+  - A named function may be made private from external modules by using `defp` instead.
+  - The value of the last line of a function is _implicitly returned_ after it is evaluated
+  - Short functions may also be written using a one-line syntax.
 
-_Named Functions_ must be defined in a module. Each [function][functions] can have zero or more parameters. All parameters are dynamically typed, and the return type is not explicitly declared, it is the type of the value returned. An _access modifier_ can be specified for functions, making only desired functions available for use external to the module. In a function, the value of the last line is _implicitly returned_ to the calling function.
+  ```elixir
+  def increment(n) do
+    n + 1
+  end
 
-```elixir
-defmodule Calculator do
-  def add(x, y) do
-    x + y
+  defp private_increment(n) do
+    n + 1
   end
-end
-```
 
-Invoking a function is done by specifying its module- and function name and passing arguments for each of the functions's parameters. The module name may be omitted if the function is invoked inside of the module.
+  def short_increment(n), do: n + 1
+  ```
+
+- Functions are invoked using the full name of the function with the module name.
+  - If invoked from within its own module, the module name may be omitted.
+- The arity of a function is often used when referring to a named function
+
+  - The arity refers to the number of parameters it accepts.
+
+  ```elixir
+  def add(x, y, z), do: x + y + z # add/3, because the arity is 3
+  ```
 
-```elixir
-sum = Calculator.add(1, 2)
-```
+## Integers
 
-If the function to be called is defined in the same module as the function that calls it, the module name can be omitted.
+- Integer values are whole numbers written with one or more digits.
+  - You may use underscores to separate large numbers.
+  - Integers support the [basic mathematical operators][operators].
 
-When referring to functions, refer to them with their _arity_. The _arity_ of a function is the number or parameters it takes.
+## Comments
 
-```elixir
-# add_3/1 because this function has one parameter, thus an arity of 1
-def add_3(x) do
-  3 + x
-end
-```
+- Elixir supports [inline documentation][inline-documentation]
 
-Elixir supports one type of comment for [inline documentation][inline-documentation]. Single line comments are preceded by `#`.
+  - Single line comments are preceded by `#`.
+  - Comments to document a named function must precede the named function definition
```

Comment:
I am not sure if I would put writing documentation with `@doc` and `@moduledoc` under the topic of "comments". In the docs, it's called "annotating" https://elixir-lang.org/getting-started/module-attributes.html#as-annotations

Maybe it makes sense to turn it around and write about "documentation", including inline comments as one of the methods? Something like in https://elixirschool.com/en/lessons/basics/documentation/

Reply:
That is true, I just didn't want to touch on module annotations just yet.  I will reorient it under the heading `Documentation` instead.